packages/components/buttons/icon-button/src/icon-button.styles.ts

```diff
@@ -99,50 +96,54 @@ const getBaseStyles = (
     return css`
       &:hover {
         background-color: ${designTokens.backgroundColorForButtonWhenHovered};
-        box-shadow: ${designTokens.shadowForButtonWhenHovered};
+        box-shadow: ${designTokens.shadow0};
       }
       ${isActive ? '&,' : ''}
       &:active {
-        background-color: ${designTokens.backgroundColorForButtonWhenActive};
-        box-shadow: ${designTokens.shadowForButtonWhenActive};
-        border-color: ${designTokens.borderColorForButtonAsIcon};
+        background-color: ${designTokens.colorSolid10};
+        box-shadow: ${designTokens.shadow0};
+        border-color: ${designTokens.colorNeutral};
       }
     `;
   }
 
   switch (theme) {
+    // TODO: These custom colors where itroduced in the Merchant Center redign 2023
+    // We need to check with designers if we want to introduce them in our color palette
+    // #15A390 -> color-primary with 10% black opacity
+    // #057FCC -> color-info with 10% black opacity
```

Comment:
I think as a basic rule, we should avoid having to hardcode such values. I would expect to use a design token for these, especially colors.

Reply:
I have a different point of view (because this is a local value), so let's discuss this in the upcoming meeting.

Reply:
I remember this was kinda of a try to have consistent `hover` and `pressed` states across the MC, without the need to introduce new color-tokens for each use case. The idea was that regardless of what color an interactive UI element has, it should become  e.g 10% darker on hover. We can also revisit this and see if there is a better approach

Reply:
Definitely this is something we will be addressing in our broader discussion about design tokens future.